Source/WebCore/platform/graphics/Latin1TextIterator.h

```diff
@@ -29,8 +29,8 @@ class Latin1TextIterator {
 public:
     // The passed in LChar pointer starts at 'currentIndex'. The iterator operates on the range [currentIndex, lastIndex].
     // 'endCharacter' denotes the maximum length of the UChar array, which might exceed 'lastIndex'.
-    Latin1TextIterator(const LChar* characters, unsigned currentIndex, unsigned lastIndex, unsigned /*endCharacter*/)
-        : m_characters(characters)
+    Latin1TextIterator(std::span<const LChar> characters, unsigned currentIndex, unsigned lastIndex, unsigned /*endCharacter*/)
```

Comment:
I don’t understand why this function takes `endCharacter`. Can we remove it?

Reply:
Will check.